src/_locales/en/messages.cson

```diff
@@ -37,7 +37,7 @@ openInNewWindow:
 editURL:
 	message: 'Download:'
 editURLWarning:
-	message: 'Caution: changing download location to different files will corrupt the download.'
+	message: 'Caution: Changing the downloads location will corrupt its files.'
```

Comment:
Thank you for your contribution. I'd like to clarifiy some items.
This line is shown when a task is being edited. Changing the location to a different URL of the same file (another mirror) is OK, while changing to a different file will corrupt it. This feature is to deal with a download site whose links frequently expire.
![image](https://user-images.githubusercontent.com/8923139/36477742-a05d3e82-1745-11e8-951c-738c3824cced.png)

Reply:
cool was unaware